HelpSource/Classes/SimpleNumber.schelp

```diff
@@ -990,3 +990,621 @@ subsection:: misc
 
 method:: isValidUGenInput
 returns:: false if receiver cannot be used in UGen.
+
+
+SECTION::Special Functions
+
+A variety of Special Functions are supplied by the Boost C++ library. The library's
+link::http://www.boost.org/doc/libs/1_66_0/libs/math/doc/html/special.html##online documentation::
+ serves as the primary reference for the following functions. The methods
+here match closely with those found in the source library, as do argument names.
+
+Below you'll find descriptions of the functions and their bounds, but for
+visualizing the functions, have a look in link::Guides/Tour-of-Special-Functions::.
+
+note::The following methods are documented in the style of a generic function call
```

Comment:
This is usually called "Functional notation" and "Receiver notation" respectively, and are described in the `Messages` help file. I'm not sure if you need this discussion here really, because everyone is (or should become) acquainted with this style variation. Were we to add it everywhere, it would be a lot of extra text in many helpfiles.

Reply:
Good point! 110ca69

Reply:
I updated the note in `SequenceableCollection.schelp` as well, though that can be removed down the line once #3694 is addressed.